install/README.md

```diff
@@ -70,6 +70,26 @@ of these settings or provide values to mandatory fields.
     - **Type:** `string`
     - :red_circle: **Mandatory!**
 
+- **`SS_AUTH_PASSWORD_MINIMUM_LENGTH`**:
+    - **Description:** sets minimum length for user passwords.
+    - **Type:** `integer`
+    - **Default:** `8`
+
+- **`SS_AUTH_PASSWORD_COMMON_VALIDATION`**:
+    - **Description:** enables password validation that prevents users from using passwords that occur in a list of common passwords.
```

Comment:
I left a note in the `base.py` module, but if we're only using these to disable rather than enable, I feel we can rename these, and make that clearer.

Reply:
Makes sense to me! That should remove the need for `is_false()` as well. I'd like to be as clear as possible to avoid confusion around the booleans (i.e. `True` turning of a validator) - how does `SS_AUTH_PASSWORD_DISABLE_COMMON_VALIDATION` sound as a name to you?

Reply:
That seems like a good approach :+1: